Core/src/main/java/com/github/intellectualsites/plotsquared/plot/commands/DebugPaste.java

```diff
@@ -115,6 +115,15 @@ private static String readFile(@NonNull final File file) throws IOException {
                         "&cSkipping PlotSquared.use_THIS.yml because it's empty");
                 }
 
+                try {
+                    final File MultiverseWorlds =
+                            new File(PlotSquared.get().IMP.getDirectory(), "../../plugins/Multiverse-Core/worlds.yml");
```

Comment:
Why is this going back two folders, to then re-enter plugins?

Reply:
Having `/plugins/Multiverse-Core/worlds.yml` does not lead to the expected result

Reply:
Getting in certain directories with Bukkit has always been a pain.

Reply:
I had no luck doing it in a better way, so I used that. If you have a better AND working solution, hmu @Sauilitired :^)

Reply:
Lol that wasn’t my point, but if you go backwards two steps you’re already leaving the plugin folder, then you directly re-enter it. What would be different by just doing ../Multiverse-Core?

Reply:
That works! I was not entirely sure how the paths behaved so I just wanted to be sure it works ^^